packages/fxa-auth-server/fxa-oauth-server/scripts/generate-client-for-ops.js

```diff
@@ -1,54 +0,0 @@
-#!/usr/bin/env node
-
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-if (process.argv.length !== 4) {
-  console.error(
```

Comment:
Has this moved elsewhere? We use this pretty regularly when creating new clients.

Reply:
moved to fxa-auth-server/scripts